qa/tasks/cephfs/test_volumes.py

```diff
@@ -645,6 +660,19 @@ def test_volume_rename_for_more_than_one_data_pool(self):
         # data pool names unchanged
         self.assertCountEqual(orig_data_pool_names, list(self.fs.data_pools.values()))
 
+    def test_rename_when_fs_is_online(self):
```

Comment:
Is this test really required since test_admin.test_rename_when_fs_is_online exists?

Reply:
In mgr/volumes, the code for renaming volumes is more than 100 LoC. IMHO, it is a nice idea to keep this test here so that we catch issues sooner.

Reply:
We discussed yesterday that the nature of this test isn't clear since we already have the corresponding test in test_admin. What specific 100 LoC is this test validating? Sorry, but this isn't an excuse to writing duplicate tests.

Reply:
Hi @vshankar. This was a mistake. I was supposed to merge "fs swap" PR and leave this one unmerged but I made a mistake while merging the batch of PRs and ended up with the reverse. You may check the fs swap PR, it's still unmerged - https://github.com/ceph/ceph/pull/50212. I'll open a PR to delete the test. 

Sorry to get you angry.  :)

Reply:
> Hi @vshankar. This was a mistake. I was supposed to merge "fs swap" PR and leave this one unmerged but I made a mistake while merging the batch of PRs and ended up with the reverse. You may check the fs swap PR, it's still unmerged - #50212. I'll open a PR to delete the test.
> 
> Sorry to get you angry. :)

I was not angry - just that I didn't find an explanation regarding the pending comments :)